sickbeard/providers/torrentz.py

```diff
@@ -90,7 +90,7 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
 
                             title_raw = item.title.text
                             # Add "-" after codec and add missing "."
-                            title = re.sub(r'([xh][ .]?264|xvid)', r'\1-', title_raw).replace(' ','.') if title_raw else ''
+                            title = re.sub(r'([xh][ .]?264|xvid) ', r'\1-', title_raw).replace(' ','.') if title_raw else ''
```

Comment:
For whitespace, i usually use \s. As that's a little better readable. It does work a bit different, because that also detects tabs, and this won't. In stead of \s you can also use [:blank:]. But both also catches tab and newline.